src/main/kotlin/io/github/gradlenexus/publishplugin/internal/NexusClient.kt

```diff
@@ -97,6 +97,28 @@ class NexusClient(private val baseUrl: URI, username: String?, password: String?
         }
     }
 
+    fun closeStagingRepository(stagingRepositoryId: String) {
+        try {
+            val response = api.closeStagingRepo(Dto(StagingRepositoryToTransit(listOf(stagingRepositoryId), "Closed by io.github.gradle-nexus.publish-plugin Gradle plugin"))).execute()
+            if (!response.isSuccessful) {
+                throw failure("close staging repository", response)
```

Comment:
Should failure message include the repository id / group id?

Reply:
I plan to make error handling for general for all the calls.